src/verify.ts

```diff
@@ -31,60 +28,42 @@ export class Verifier {
     this.rekor = options.rekor;
   }
 
-  public async verify(
-    payload: Buffer,
-    signature: string,
-    certificate: KeyLike
-  ): Promise<boolean> {
-    signature = signature.trim();
+  public async verify(bundle: Bundle, data?: Buffer): Promise<boolean> {
```

Comment:
Looking at @kommendorkapten's [bundle PR](https://github.com/sigstore/cosign/pull/2204) there's some logic to defer [verification to a passed in verifier function](https://github.com/kommendorkapten/cosign/blob/29f29f284e59f09a5b562c2d2285688734a98995/specs/dotsigstore_bundle/verifier.go#L65-L71).

I think the thinking here is to avoid potential key-confusion attacks so maybe not relevant for us (yet) as we're only verifying using the bundled cert?